### PR TITLE
Update sample config with livekit (rebase on livekit)

### PR DESCRIPTION
**Pull request by toger5**
replaces: https://github.com/element-hq/element-call/pull/1212

### CODE REVIEW

**Comment by fkwp** [comment]
do we need this key? going forward the information will be provided by .well-known

**Comment by toger5** [comment]
I think for now its good since its still in the readme. Nowadays the best would be to properly updated the readme and explain everything.

**Comment by toger5** [comment]
Just checked this is already explained in the readme: https://github.com/element-hq/element-call/blob/d0e848d90d8c29c868fa4bd0b0478554e41878a4/README.md#L127-L134

so this really just brings the example up to speed

File: config/config.sample.json [hunk 1]
```diff
@@ -5,5 +5,8 @@
       "server_name": "call.ems.host"
     }
   },
+  "livekit": {
+    "livekit_service_url": "http://localhost:7881"
```
Comment (AndrewFerr) [inline config/config.sample.json:9]: How about we update the port in the README example to match this?

**Review by AndrewFerr — APPROVED** [review]

**Pull request merged**